components/Pages/RoomIdPage.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import Cookies from 'js-cookie';
+
+const backendApiUrl = 'http://localhost:5000';
+
+class RoomIdPage extends React.Component {

```

Comment:
can consider swapping to function component syntax `export default function RoomIdPage(props) {`
and setting state becomes like `const [roomId, setRoomId] = React.useState(props.roomId);` etc